### PR TITLE
Add Privacy Consideration section

### PR DESCRIPTION
**Pull request by QingAn**
***
<a href="https://pr-preview.s3.amazonaws.com/QingAn/miniapp-lifecycle/pull/19.html" title="Last updated on Sep 24, 2021, 10:03 AM UTC (80cc5ca)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/miniapp-lifecycle/19/13c17b5...QingAn:80cc5ca.html" title="Last updated on Sep 24, 2021, 10:03 AM UTC (80cc5ca)">Diff</a>

***

### :boom: Error: 500 Internal Server Error :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on May 28, 2022, 2:31 AM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [Spec Generator](https://www.w3.org/2015/labs/) - Spec Generator is the web service used to build specs that rely on ReSpec.

:link: [Related URL](https://labs.w3.org/spec-generator/?type=respec&url=https%3A%2F%2Fraw.githubusercontent.com%2FQingAn%2Fminiapp-lifecycle%2F08623fdc371c1fc776ecfdd6e7f5ac478c4325d0%2Findex.html%3FisPreview%3Dtrue)

```
📡 HTTP Error 404: http://labs.w3.org/spec-generator/uploads/Kz6VO5
```

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20w3c/miniapp-lifecycle%2319.)._
</details>

### CODE REVIEW

**Comment by QingAn** [comment]
@Sharonzd, @MichaelWangzitao 

Updated as we discussed during the meeting. Please take a look.

File: index.html [hunk 1]
```diff
@@ -643,6 +643,16 @@ <h2>Privacy and Security</h2>
          
      </section>
     
+     <section class='informative'>
+      <h2>Privacy Considerations</h2>
+      <p>
+          MiniApp Global Application Lifecycle interface {{Global}} introduces {{inputObject}} involves the processing of the inputted query for the MiniApp, the page path for current MiniApp, and the source info. MiniApp Page Lifecycle interface {{Page}} introduces {{pageInputObject}} involves the processing of the inputted query for the MiniApp page. In order to protect users from any potential unsanctioned tracking threat, it is not recommended to store these values locally. If these values are stored, these storages should be cleared when users intend to clear them.
+      </p>
+      <p>
+         If the inputted query for the MiniApp or the inputted query for the MiniApp page contains privacy-sensitive information (e.g. user personal data), the privacy-sensitive information shall not be in plaintext format.
```
Comment (xfq) [inline index.html]:
```suggestion
         If the inputted query for the MiniApp or the inputted query for the MiniApp page contains privacy-sensitive information (e.g. user personal data), the privacy-sensitive information shall not be in cleartext.
```

[Cleartext](https://www.hypr.com/cleartext/) should be better here.

**Comment by Sharonzd** [comment]
It looks good to me.

**Comment by xfq** [comment]
Looks good to me as a preliminary draft. We can make further adjustments if needed.

**Comment by xfq** [comment]
Merging.

**Comment by MichaelWangzitao** [comment]
Looks good to me, as well.

**Pull request merged**